recipes/doebase/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "doebase" %}
+{% set version = "v2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pablocarb/doebase/archive/refs/tags/{{ version }}.tar.gz
+  sha256: a0cc4af91a42c8c92ee901947a6855cfcb999b21bdde50f0869b7c7c52557a7d
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install -vv .
+  binary_relocation: true
+  missing_dso_whitelist:
+    - /lib64/libc.so.6
+    - $RPATH/libxml2-8ed442f7.so.2.6.26
+    - $RPATH/libz-a147dcb0.so.1.2.3
+    - /lib64/libm.so.6
+    - /lib64/libdl.so.2
+    - lib/libstdc++.so.6
+    - lib/libgcc_s.so.1
```

Comment:
Why are these needed?

Reply:
Hi @ocefpaf  ,

this is the kind of errors returned by conda that @breakthewall told you about:

```
2021-11-24T13:30:34.8922236Z   ERROR (pysbol,lib/python3.7/site-packages/pysbol/lib/Linux/.libs/libxml2-8ed442f7.so.2.6.26): /lib64/libm.so.6 not found in packages, sysroot(s) nor the missing_dso_whitelist.
2021-11-24T13:30:34.8922861Z .. is this binary repackaging?
```
We could avoid it by adding these libraries in the **missing_dso_whitelist**. And we proceeded in the same way for these 2 other packages : https://github.com/conda-forge/staged-recipes/pull/17106  https://github.com/conda-forge/staged-recipes/pull/16673 .

Thank you for your work.